lib/dogapi/facade.rb

```diff
@@ -296,6 +297,29 @@ def graph_snapshot(metric_query, start_ts, end_ts, event_query = nil)
     end
 
     #
+    # EMBEDS
+    #
+    def get_all_embeds()
+      @embed_svc.get_all_embeds()
+    end
+
+    def get_embed(embed_id, size="medium", legend="no", template_vars= {})
+      @embed_svc.get_embed(embed_id, size, legend, template_vars)
+    end
+
+    def create_embed(graph_json, timeframe="1_hour", size="medium", legend="no", title="Embed created through API")
```

Comment:
Well titles are required in the sense that the embed has to have a title at least 1 character in length and the default title is `"Embed created through API"`